zkevm-circuits/src/keccak_circuit.rs

```diff
@@ -101,7 +104,7 @@ impl<F: Field> SubCircuitConfig<F> for KeccakCircuitConfig<F> {
         let is_final = keccak_table.is_enabled;
         let length = keccak_table.input_len;
         let data_rlc = keccak_table.input_rlc;
-        let hash_rlc = keccak_table.output_rlc;
+        let hash_word = keccak_table.output;
```

Comment:
very nitpick: `hash_word_le` might be self-explanation since hash usually mean big endian.

Reply:
I'm not sure if I understand. Do you suggest that we rename `hash_word` to `hash_word_le`?

I think keeping "hash_word" as it is is good because it represents the final output. We don't have to worry about the endianness for it.

Reply:
haha, I think "output_word" might be better, because "hash" used to imply big ending. but it's ok to keep current hash_word for now